api/src/pcapi/routes/public/individual_offers/v1/serialization.py

```diff
@@ -943,3 +947,25 @@ class Config:
 
 class GetOfferersVenuesQuery(serialization.ConfiguredBaseModel):
     siren: str | None = pydantic.Field(example="123456789")
+
+
+class GetProductsListByEansQuery(serialization.ConfiguredBaseModel):
+    eans: str | None = pydantic.Field(example="0123456789123,0123456789124")
+
+    @pydantic.validator("eans")
+    def validate_ean_list(cls, eans: str) -> list:
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏`eans: list[str]`

Reply:
En entrée on s'attend à un str qui ressemble à 1111111111111,1111111111111...
Et on le change en list manuellement
Si je mets list[str] comme type de eans la route casse avec une validation error